### PR TITLE
Adds scripts to the specific entities counted.

### PR DESCRIPTION
**Pull request by cerebrate**
Nice to be able to retire my old automations for counting entities by type now that Spook is here, but I noticed that a count of scripts wasn't included.

Now it is!

#overcompensation

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
Looks good!

Could you please update the README to list it as well? _#NobodyLikesDoingDocs_

../Frenck

**Comment by cerebrate** [comment]
Done!

File: README.md [hunk 1]
```diff
@@ -168,7 +169,7 @@ of the box.
 # Services
 
 There are quite a few useless and horrible services available for you to explore
-and self-descruct your setup with. The developer service tools are great
```
Comment (frenck) [inline README.md:171]: Thanks! 👀

**Review by frenck — APPROVED** [review]
Thanks, @cerebrate 👍

../Frenck

**Pull request merged**